src/platform/silabs/efr32/BLEManagerImpl.cpp

```diff
@@ -554,8 +564,16 @@ CHIP_ERROR BLEManagerImpl::StartAdvertising(void)
     }
     else
     {
-        interval_min = CHIP_DEVICE_CONFIG_BLE_SLOW_ADVERTISING_INTERVAL_MIN;
-        interval_max = CHIP_DEVICE_CONFIG_BLE_SLOW_ADVERTISING_INTERVAL_MAX;
+        if (!(mFlags.Has(Flags::kExtAdvertisingEnabled))) {
```

Comment:
If we remove the else we should also remove the if imo

Reply:
Sure, I did not want to add another, but yes its nicer.